main.js

```diff
@@ -1,53 +1,83 @@
-const bookList = document.querySelector('.books');
-const title = document.getElementById('title');
-const author = document.getElementById('author');
-const addBook = document.querySelector('#add-books');
-
-const Book = function theBook(title, author) {
-  this.title = title;
-  this.author = author;
-};
-
-let storedData = [];
-
-function addBookToList(newBook) {
-  const bookTemp = `
-    <div class="book">
-      <h2>Title: ${newBook.title}</h2>
-      <h2>Author: ${newBook.author}</h2>
-      <button class="delete" type="button">Remove</button>
-      <hr>
-    </div>
-  `;
-  document.querySelector('.books').innerHTML += bookTemp;
-  return document.querySelector('.books').innerHTML;
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
 
-document.addEventListener('DOMContentLoaded', () => {
-  if (bookList !== null) {
-    storedData = [...JSON.parse(localStorage.getItem('localBookList'))];
-    storedData.forEach((item) => {
-      addBookToList(item);
+class Displays {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Displays.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(author) {
+    const books = Displays.getBooks();
+    books.forEach((book, index) => {
+      if (book.author === author) {
+        books.splice(index, 1);
+      }
     });
+    localStorage.setItem('books', JSON.stringify(books));
   }
```

Comment:
- [ ] Great job on refactoring your code 👍 however after adding a book series with the same author, for example, Harry Potter, J.K. Rowling, when the user deletes a book first it deletes the right book, however after reload it will always be the first book in the list written by that author. this is because you are removing the book from localStorage based on the author name and it will always pick the first instance of that name in your list. in order to fix this you might want to consider implementing a unique id for every book when it is added and deleting it based on the id number.

Reply:
Done